patches/tModLoader/Terraria/ModLoader/ModSoundStyle.cs

```diff
@@ -0,0 +1,45 @@
+using Microsoft.Xna.Framework.Audio;
+using System.IO;
+using Terraria.Audio;
+
+namespace Terraria.ModLoader
+{
+	public class ModSoundStyle : SoundStyle
```

Comment:
Just a minor comment: implement existing IModType?
Both ModTileEntity and This derive a vanilla base, so would be good to see consistency across the two in use of that interface.

Reply:
This is a struct-like class, unrelated to ModTileEntity.